spring-boot-project/spring-boot/src/main/java/org/springframework/boot/BootstrapContext.java

```diff
@@ -77,7 +77,7 @@ public interface BootstrapContext {
 	<T, X extends Throwable> T getOrElseThrow(Class<T> type, Supplier<? extends X> exceptionSupplier) throws X;
 
 	/**
-	 * Return if a registration exists for the given type.
+	 * Return {@code true} if a registration exists for the given type.
```

Comment:
While we aren't consistent across the codebase, I don't consider this one to be a typo or grammatical error. We should leave it as is for now please.

Reply:
No need to do anything here. I dropped this change while merging the PR.